config/events/events.cwt

```diff
@@ -175,8 +175,21 @@ enums = {
     enum[event_option_skills] = {
         learning
     }
-    enum[event_themes] = {
-        feast
-        romance
+    complex_enum[event_themes] = {
```

Comment:
This can probably better be done as a type.
A type provides equivalent (if not richer) features than a complex_enum when they can be used. Complex enums are best for weird nested things.
Type here would literally be something like:
```
types = {
    type[event_theme] = {
        path = "game/common/event_themes"
    }
}
```

Reply:
Great. I'll open an issue for a refactor to this to track it.